src/NServiceBus.AcceptanceTests/MessageId/When_message_has_no_id_header.cs

```diff
@@ -5,7 +5,7 @@
     using System.Threading.Tasks;
     using AcceptanceTesting;
     using EndpointTemplates;
-    using Pipeline;
+    using NServiceBus.Pipeline;
```

Comment:
this seems to be required because I added a `Pipeline` folder to the acceptance tests